build.gradle

```
@@ -30,7 +30,7 @@ java {
 }
 
 group = "io.kestra.plugin"
-description = 'Plugin template for Kestra'
+description = 'Plugin Influxdb for Kestra'
```

```suggestion
description = 'Plugin InfluxDB for Kestra'
```

Please check everywhere that it's correctly spelled as InfluDB.